index.html

```diff
@@ -40,3 +40,166 @@ <h2 id="connect">LET'S CONNECT</h2>
 </ul>
                 
     </section>
+    <section id ="my-works">
+    <div class="all-content">
+        
+            <img src="media/Snapshoot Portfolio.png">
+        
+        <h1 id ="project-title">
+            Tonic
+        </h1>
```

Comment:
- [ OPTIONAL]   Please use the `hx` tags semantically. Your main page heading is already `h1`.
```
  <div id="greeting">
        <h1>I'm John Glad to see you</h1>
    </div>
```
For your project headings, it would be good to give then `h2, h3,..`. Not that all heading at the same level should be given the same `hx` for example once you decide to give `h2` to a project titles, all project title should have h2  and not `h3, h4,,..`

consider changing to 
```
<h2 id ="project-title">
            Tonic
        </h2>
```